docs/users/cli-usage.md

```diff
@@ -163,6 +193,11 @@ ddev import-db --src=dumpfile.sql.gz
 
 For in-depth application monitoring, use the `ddev describe` command to see details about the status of your ddev app.
 
+**Note for Backdrop users:** In addition to importing a Backdrop database, you will need to put your Backdrop site's `active` configuration into `files/ddev_config/active`. Without these config files present, Backdrop will not run properly. If `files/ddev_config` is not where your project stores its config, you can replace the `ddev_config` directory with a symlink to whereever you store your config. An example of this command is as follows:
```

Comment:
I'm imagining this might be a good warning to print out when a project of apptype "backdrop" runs an import-db command. As a Drupal or WordPress end-user, I might not expect to hit upon this item and if I didn't read the documentation ahead of time, I would be dead in the water.